presto-hive/src/main/java/com/facebook/presto/hive/HiveTableLayoutHandle.java

```diff
@@ -210,6 +213,16 @@ public String getLayoutString()
         return requestedColumns;
     }
 
+    public void withDynamicFilterPredicate(TupleDomain<ColumnHandle> dynamicFilterPredicate)
+    {
+        this.dynamicFilterPredicate = dynamicFilterPredicate;
+    }
+
+    public TupleDomain<ColumnHandle> getDynamicFilterPredicate()
+    {
+        return dynamicFilterPredicate;
+    }
```

Comment:
As discussed offline yesterday. We should put `dynamicFilterPredicate` in SplitContext rather than creating new interfaces in SPI.